src/open_producten/producttypen/migrations/0007_externecode.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 4.2.17 on 2025-02-18 16:17
+
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("producttypen", "0006_contentelement_contentelementtranslation_and_more"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ExterneCode",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                (
+                    "naam",
+                    models.CharField(
+                        help_text="De naam van het systeem van de externe product code.",
+                        max_length=255,
+                        validators=[
+                            django.core.validators.RegexValidator("^[^:\\[\\]]+$")
```

Comment:
The regex here seems different from the one specified in the models: `^[^:\[\]]+$` :thinking:

Reply:
it's because of the `r` string inside the models. 
![image](https://github.com/user-attachments/assets/6f59416d-177b-4f86-a777-b58746344f30)